srcpkgs/starship/template

```diff
@@ -1,18 +1,19 @@
 # Template file for 'starship'
 pkgname=starship
-version=0.51.0
+version=0.55.0
 revision=1
 build_style=cargo
 build_helper=qemu
-hostmakedepends="pkg-config libgit2-devel"
-makedepends="openssl-devel libgit2-devel"
+hostmakedepends="libgit2-devel pkg-config"
+makedepends="libgit2-devel openssl-devel"
+depends="git"
```

Comment:
Ugh, they now depend on both `libgit2` *and* the git command line?

Reply:
Unfortunately yes. The git cli is required for their rewritten git module, but for some reason libgit2 is still listed as a requirement in their `cargo.toml` file. I tried building without `libgit2`; it doesn't work.